packages/card/src/Card.ts

```diff
@@ -78,6 +78,9 @@ export class Card extends LikeAnchor(
     set selected(selected: boolean) {
         if (selected === this.selected) return;
         this._selected = selected;
+        if (this.role === 'row' && this.toggles) {
```

Comment:
@majornista can you add some a11y notes and examples for this in `packages/card/EADMEN.md`?

Reply:
I've added documentation for the `toggles` property in `packages/card/README.md`.  This code addresses how the selected state for a card gets communicated when it is used as a `row` item within a `grid`.